### ISSUE TITLE
Breakpoints are reactivated across VS Code launches when using "Deactivate Breakpoints"

### ISSUE THREAD

**Issue by ivanhernandez13** [microsoft/vscode#88049]
Issue Type: <b>Bug</b>

1. git clone https://github.com/microsoft/vscode-mock-debug.git
2. Open mock debug and start debugging.
3. Open any markdown file in the debuggee VS Code process and set a breakpoint anywhere.
4. Deactivate all breakpoints using the button in the Breakpoints title bar.
5. Reload window

This option is not remembered across sessions. Disabling each individual breakpoint works as expected and is respected across sessions.

VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T14:57:51.166Z)
OS version: Darwin x64 19.2.0

**Comment by isidorn** [comment]
Yes, I confirm this behavior and this is by design.
Reason being: we designed breakpoint deactivation as something temporary and easy to get out of. As soon as you add a breakpoint all breakpoints get activated and we also decided to not preserve this across vscode sessions.
No one has complained for 4 years now, however I will leave this feature request open and I open to reconsider the original design of this and that we start preserving deactivation across vscode sessions.

Thanks!
